velox/vector/arrow/Bridge.h

```diff
@@ -80,6 +80,9 @@ void exportToArrow(
 /// actual data (containing encoding) to create an ArrowSchema.
 void exportToArrow(const VectorPtr&, ArrowSchema&);
 
+/// Export the type of a Velox TypePtr to an ArrowSchema.
+void exportToArrow(const TypePtr&, ArrowSchema&);
```

Comment:
This is omitted intentionally because Arrow type need both Velox type and encoding.  Can you use `exportToArrow(BaseVector::create(type, 0, pool), out)` if encoding is assumed to be all flat here?

Reply:
@Yuhta @majetideepak @mbasmanova Therefore, there is no need to include the export API, correct?

Reply:
@Yuhta Jimmy, would you clarify a bit? It seems inefficient to create a vector when all we need is type + encoding. Why can't we add exportToArrow(type, encoding, schema&) or exportToArrowFlat(type, schem&) API?

Reply:
@mbasmanova encoding is not a single value but a tree, so we need an empty vector to serve as container for it.

Reply:
@JkSelf Yes this is omitted intentionally until arrow decouple encoding from type

Reply:
@Yuhta Interesting. Is there a GitHub in the arrow repo about this? If not, let's create one and add a comment here to explain the limitation and how to go around it.

Reply:
@mbasmanova Looks like there is no issue in the repo.  Proposals like this are discussed over mailing list in Arrow community as far as I know.  We raised this issue to @wesm last year during a meeting and he agreed this is a mistake in design, and hopefully can be fixed in the next breaking format change of Arrow.  The timeline for that is unclear though, since they are keeping adding new encoding (RLE) as new type for now.

@pedroerp @kgpai Do you guys have any more information on this arrow encoding type decoupling topic?

Reply:
Is it possible to add link to mailing list discussion?

Reply:
@mbasmanova Here it is: https://lists.apache.org/thread/pmv0bx4jog6kbz9f2o2yp5x4k0twz1oc

Reply:
I don't know if there has been any progress on this in the intervening 14 months (I started the discussion, but had to turn my attention to other things and wasn't able to push it forward myself). 

I think it would be possible to introduce encoded arrays in a forward compatible manner (i.e. existing C ABI readers and IPC / binary protocol readers would at least be able to recognize the presence of data they can't read and not break unpredictably) but what's missing is the design work to put forward a proposal of what exactly are those new metadata structures and other memory layout details to send encoded data over the wire or expose it via the C ABI. This seems very doable to me but needs a "champion" to move the project forward.

Reply:
I guess in the short term the only thing we could do, if we want to provide this API, is to assume the vectors/arrays will be flat encoded (or maybe take a parameter where users can specify the encoding)? Though I agree it is a bit awkward and error prone.

Reply:
The need for this PR can be solved by other means. We don't need this change at this time.
More details are here https://github.com/facebookincubator/velox/pull/6074

Reply:
@majetideepak Sure. I will close this PR. Thanks for all your help.